src/main/java/com/coveo/saml/SamlClient.java

```diff
@@ -95,7 +93,7 @@ public void setDateTimeNow(DateTime now) {
    *                                    SAML response.
    * @param identityProviderUrl         the url where the SAML request will be submitted.
    * @param responseIssuer              the expected issuer ID for SAML responses.
-   * @param certificate                 the base-64 encoded certificate to use to validate
+   * @param certificates                the list of base-64 encoded certificates to use to validate
```

Comment:
It would be nice to provide an overload that takes a single certificate for backward compatibility.